cloudformation/cmp_aus.yaml

```diff
@@ -0,0 +1,117 @@
+Parameters:
```

Comment:
This is probably a for a follow up but is there any way we could generate these templates rather than having three separate ones?

Reply:
This will be solved by using CDK. We'll also get extravagances like variables and loops which will make it easier to read with less duplication.